codetransformer/instructions.py

```diff
@@ -64,7 +70,7 @@ def _vartype(self):
 
 
 class InstructionMeta(ABCMeta, matchable):
-    _marker = object()  # sentinel
+    _marker = type('_marker', (object,), {})  # sentinel
```

Comment:
I thought it might be causing the issues we were seeing with Instruction not being discovered correctly by sphinx, but that wasn't the case.

I left it because it seems slightly odd to have a random non-type object in Instruction's mro.